### PR TITLE
Adding Bighorners sprites and code -Might need unfucking-

### PR DESCRIPTION
**Pull request by GoodG1448**
Look at #630

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -395,6 +395,45 @@
...
+	melee_damage_lower = 25
+	melee_damage_upper = 20
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	stop_automated_movement_when_pulled = 1
+	blood_volume = BLOOD_VOLUME_NORMAL
+
+/mob/living/simple_animal/hostile/retaliate/goat/bighorn
+	name = "bighoner lamb"
+	resize = 0.55
+	is_calf = 1
```
Comment (improvedname) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: code/modules/mob/living/simple_animal/friendly/farm_animals.dm:435:error: is_calf: undefined var

**Comment by improvedname** [comment]
Something more like this

**Comment by GoodG1448** [comment]
I guess that is it. Thanks

**Comment by GhostEcho** [comment]
So issue with this, the big horners attack each other

**Pull request merged**